common/metrics/tags.go

```diff
@@ -383,3 +400,44 @@ func DestinationTag(value string) Tag {
 		value: value,
 	}
 }
+
+func VersioningBehaviorBeforeOverrideTag(behavior enumspb.VersioningBehavior) Tag {
+	return &tagImpl{key: behaviorBefore, value: behavior.String()}
+}
+
+func VersioningBehaviorAfterOverrideTag(behavior enumspb.VersioningBehavior) Tag {
+	return &tagImpl{key: behaviorAfter, value: behavior.String()}
+}
+
+func RunInitiatorTag(prevRunID string, attributes *historypb.WorkflowExecutionStartedEventAttributes) Tag {
+	// UpdateWorkflowExecutionOptions API is the caller
+	if attributes == nil {
+		return &tagImpl{key: runInitiator, value: existingRun}
+	}
+
+	// StartWorkflowExecution API is the caller
+	if prevRunID == "" {
```

Comment:
do you really need to pass prevRunID? doesn't CONTINUE_AS_NEW_INITIATOR_UNSPECIFIED mean new?